tests/deepsparse/v2/schedulers/test_continuous_batching_scheduler.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from concurrent.futures import Future
+
+import numpy
+
+from deepsparse.v2.operators import EngineOperator
+from deepsparse.v2.schedulers import ContinuousBatchingScheduler
+
+
+def test_continuous_batching_executor_thread():
+    # simple test that ContinuousBatchingScheduler can be instantiated and return
+    # a result from a request, for testing multi-batch execution, making enough
+    # concurrent requests guarantee batched execution is out of scope
+    scheduler = ContinuousBatchingScheduler()
```

Comment:
Is the goal that we would initialize (at least) two schedulers, one `ContinuousBatchingScheduler` and the remaining `OperatorSchedulers` as part of the pipeline init? And then swap to use `ContinuousBatchingScheduler` when the operator being executed is of type `EngineOperator`?

Reply:
yes exactly, the `ContinuousBatchingScheduler` would come first in the list of schedulers so it can pick up any relevant engine operator jobs